### DISCUSSION TITLE
Watchman Report

### DISCUSSION THREAD

**Discussion by adamengineering** [Blackymas/NSPanel_HA_Blueprint#918] [Q&A] [answered]
In my report there are 2 missing entities:

`switch.turn_          | missing | blueprints/automation/Blackymas/nspanel_blueprint.yaml:6192, 6196`
`climate.climate_label | missing | blueprints/automation/Blackymas/nspanel_blueprint.yaml:7457`

This missing entities aren't causing me any obvious problem, but I would like to know, what they would be for?

**Comment by edwardtfn** [comment, marked as answer]
`switch.turn_` is part of a constructor for a service call to turn the switches on or off when you press a button. Watchman interprets as an invalid service, but it's not the service call used by the blueprint.
`climate.climate_label` is a reference for a component named `climate_label` in the page `climate`. This is used in a string used in a call to a ESPHome service to update that component in the scree and this have nothing to do with an entity. Watchman probably see this as an entity due to the name starting like `climate.`, but this is not correct interpretation from Watchman.

I would suggest to exclude that file related from this Blueprint from your Watchman report. This is what I did on my side. 😉

**Comment by primeroz** [comment]
I have the same problem , i just exclude the `whole` nspanel blueprint from watchman by ignoring the file `/config/blueprints/automation/Blackymas/nspanel_blueprint.yaml`
